LayoutTests/platform/ios/svg/text/font-small-enlarged-minimum-larger-expected.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="width: 400px; height: 400px;">
```

Comment:
I never added a platform specific reftest ..-expected.svg/html file, that deviates from the normal result. Usually we make platform exceptions for txt dumps / pngs.

You need to describe why you need/want th in the commit message -- this cannot be understood without more context. :-(